tests/mocks/lms/djangoapps/certificates/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.15 on 2019-03-23 20:18
+# Generated by Django 1.11.15 on 2019-07-26 02:19
```

Comment:
I had to regenerate the migrations for Hawthorn model changes to `CourseEnrollment`. These migrations are required for the tests. This is a change in Django 1.9